chart/epinio-ui/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 annotations:
   artifacthub.io/license: Apache-2.0
 apiVersion: v2
-appVersion: v0.6.2
+appVersion: v0.7.1
```

Comment:
I am a bit confused, is it supposed to be `v0.7.1-0.0.2`?

Reply:
v0.7.1-0.0.2 is the latest tag published on https://github.com/epinio/ui-backend/tags

Reply:
You're right, that should have been the explicit ui build number. The `epinioVersion` in values.yaml below is what we display to the user which we've agreed will track the core epinio app number. There's no harm as is though, the next bump will be semver newer